features/fixtures/rails3/app/app/controllers/session_tracking_controller.rb

```diff
@@ -6,15 +6,19 @@ def index
   end
 
   def initializer
-    Bugsnag.session_tracker.send_sessions()
+    Bugsnag.session_tracker.send_sessions
     render json: {}
   end
 
-  def after
-    Bugsnag.configure do |conf|
-      conf.auto_capture_sessions = true
-    end
-    Bugsnag.session_tracker.send_sessions()
+  def manual
+    Bugsnag.start_session
+    Bugsnag.session_tracker.send_sessions
+    render json: {}
+  end
+
+  def hundred
```

Comment:
I would name this `many` or `multiple` or something like that so the name still makes sense if you change the number of sessions started.